concord-server/server/src/main/java/com/rabidgremlin/concord/resources/StatsResource.java

```diff
@@ -74,10 +86,42 @@ public Response getUserStats(@ApiParam(hidden = true) @Auth Caller caller)
           int totalWithConsensusIgnoringTrash = getVotesForUser(totalVotesForPhrasesWithConsensusIgnoringTrash, userId);
           return new UserStats(userId, total, completed, trashed, totalWithConsensus, completedIgnoringTrash, totalWithConsensusIgnoringTrash);
         })
-        .sorted(Comparator.comparing(UserStats::getTotalVotes).reversed())
-        .collect(Collectors.toList());
+        .collect(toList());
 
     return Response.ok().entity(userStats).build();
   }
 
+  private int getVotesForUser(List<UserVoteCount> list, String userId)
+  {
+    return list.stream()
+        .filter(userVoteCount -> userVoteCount.getUserId().equals(userId))
+        .findFirst()
+        .map(UserVoteCount::getVoteCount)
+        .orElse(0);
+  }
+
+  @GET
+  @Timed
+  @Path("/system")
+  public Response getSystemStats(@ApiParam(hidden = true) @Auth Caller caller)
+  {
+    log.info("{} getting system stats.", caller);
+
+    int totalPhrases = systemStatsDao.getTotalCountOfPhrases();
+    int completedPhrases = systemStatsDao.getCountOfCompletedPhrases();
+    int labelsUsed = systemStatsDao.getCountOfLabelsUsed();
+    int totalVotes = systemStatsDao.getCountOfVotes();
+    int totalLabels = systemStatsDao.getCountOfLabels();
+    int userCount = systemStatsDao.getCountOfUsers();
+
+    List<GroupedPhraseVoteWithMostRecentVoteTime> votedLabelsForUncompletedPhrases = votesDao.getLabelsForUncompletedPhrasesInVoteCountOrder();
+    Set<String> phraseIdsVotedOnByGodUser = votesDao.getVotesMadeByUser(RESOLVER_USER_ID).stream().map(PhraseLabel::getPhraseId).collect(Collectors.toSet());
```

Comment:
Minor: Probably good to rename it to phraseIdsVotedOnByResolver to make it consitent